nav2_bringup/package.xml

```diff
@@ -15,9 +15,13 @@
   <build_depend>navigation2</build_depend>
   <build_depend>launch_ros</build_depend>
 
+  <exec_depend>diff_drive_controller</exec_depend>
+  <exec_depend>joint_state_broadcaster</exec_depend>
   <exec_depend>launch_ros</exec_depend>
   <exec_depend>navigation2</exec_depend>
   <exec_depend>nav2_common</exec_depend>
+  <exec_depend>ros_gz_bridge</exec_depend>
```

Comment:
Add xacro

Reply:
Otherwise it crashes

Reply:
Added in https://github.com/ros-planning/navigation2/pull/3634/commits/6a462beb7b5d51992a66490739c10eb2fc3f2134